README.md

```diff
@@ -29,6 +29,11 @@ Git repository.
 - `POLL_INTERVAL_SECONDS` - (int) (default: `5`) Number of seconds to wait between each check for new commits to the repo
 - `REPO_PATH_FILTERS` - (string) (default: `""`) A comma separated list of sub directories to be applied. Supports [shell file name patterns](https://golang.org/pkg/path/filepath/#Match).
 
+#### Variables needed by terraform resources
+
+Appart from the variables needed to run terraform-applier, you also need to provide all the variables needed by the used terraform providers
```

Comment:
`Apart`*

Reply:
Also:
> you also need to provide all the variables needed by the used terraform providers 

isn't strictly true. You don't need to provide all of them or in some cases any of them. These values can live in the code with the provider declaration. 

It might be more correct to say:
> You can also provide environment variables for use by terraform providers (such as `AWS_ACCESS_KEY_ID`) or variables for use in your code (`TF_VAR_your_variable_name`) . This is useful for providing sensitive values that you don't want to save in version control or variables that are only available in your Kube environment.